common/src/main/java/com/wfsample/common/BeachshirtsUtils.java

```diff
@@ -0,0 +1,66 @@
+package com.wfsample.common;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.fasterxml.jackson.dataformat.yaml.YAMLParser;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Map;
+
+import javax.annotation.Nullable;
+
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+/**
+ * Utilities for use by the various beachshirts application related services.
+ *
+ * @author Srujan Narkedamalli (snarkedamall@wavefront.com).
+ */
+public final class BeachshirtsUtils {
+
+  private BeachshirtsUtils() {
+  }
+
+  public static <V> V httpGet(OkHttpClient client, String host, int port, String pathSegments,
```

Comment:
Please change from OkHttpClient to RestEasyClient.
Let's do this now before adding any SDKs.